pinot-common/src/main/java/org/apache/pinot/common/config/TableConfig.java

```diff
@@ -157,7 +168,7 @@ public static TableConfig fromJsonConfig(JsonNode jsonConfig)
     if (childConfigNode == null || childConfigNode.isNull()) {
       return null;
     }
-    if (childConfigNode.isObject()) {
+    if (childConfigNode.isObject() || childConfigNode instanceof ArrayNode) {
```

Comment:
Equivalent to `childConfigNode.isContainerNode()`

Reply:
Done